### DISCUSSION TITLE
Configure Snipe-It Pre-Flight through CLI

### DISCUSSION THREAD

**Discussion by kadattack** [snipe/snipe-it#15796] [Q&A] [answered]
I can't really find documentation on how to configure Snipe-It through CLI, since i want to do it through a script.

After installing Snipe-It, I create the database `snipeit` and a superuser that will handle migration. I put the info in to the .env file for login data that snipeit will use
```
MYSQL_PORT_3306_TCP_ADDR={{ host_machine }}
MYSQL_PORT_3306_TCP_PORT=3306
MYSQL_DATABASE={{ snipeit_db_name }}
MYSQL_USER={{ MySQL_Username }}
MYSQL_PASSWORD={{ MySQL_Password }}
```
I then run `APP_ENV=env_file.env php artisan migrate --force`
and then follow it with 
`php artisan snipeit:create-admin --first_name={{ admin_name }} --last_name={{ admin_name }} --email={{ admin_login_email }}  --username={{ admin_login_email }}  --password={{ admin_password }}"`

I then visit the Snap-It website http://192.168.122.31/ and i keep get redirected to http://192.168.122.31/setup which is the Snipe-IT Pre-Flight page. I thought the point of migrating and creating an admin account was to avoid doing it again through the web page.

I've checked the database and the tables to seem to have been created with the `php artisan migrate`. 

How do I bypass having to redo everything on the /setup page all over again?

**Comment by snipe** [comment, marked as answer]
You'd need to add a record in the settings table. The system forwards to setup if there is nothing in the `settings` table and also no user records.
